imagestreams/ruby-rhel7.json

```diff
@@ -22,7 +22,27 @@
         },
         "from": {
           "kind": "ImageStreamTag",
-          "name": "2.7"
+          "name": "2.7-el7"
+        },
+        "referencePolicy": {
+          "type": "Local"
+        }
+      },
+      {
+        "name": "2.7-el7",
```

Comment:
Why is it `2.7-el7`?

Do I understand correctly that it's because there's no `2.7-el8` yet, or is there any other preference?

Reply:
What I meant is why is this marked as latest.

Reply:
ruby:2.7-el7 => rhscl/ruby-2.7-rhel7 for OCP, centos/ruby-2.7-centos7 for OKD

Reply:
There is no 2.7-ubi8 => ubi8/ruby-27 released yet.  Once that is released, that should be latest.

Reply:
Yes, right.
I just want to be clear that  `ubi7` and `ubi8` are not completely interchangable. So previously this imagestream was `-rhel7`, which is why it's ok too set `-el7` as latest __now__, but once we switch to `ubi8` (some/most users will need to manually update their apps), we'll need to keep `-ubi8` _always_ as "latest" (or some `-ubi9` version f.e.).